### DISCUSSION TITLE
m1芯片的配置教程

### DISCUSSION THREAD

**Discussion by leotongxue** [Yang-Xijie/yang-xijie.github.io#10] [Cpp in VSCode on macOS]
期待可以出一版m1芯片的教程

**Comment by Yang-Xijie** [comment]
我现在手头没有M1的机器 T_T
有了会更新教程的ww

或者大家有类似的教程推荐，我也可以在我对应的教程下方置顶写出M1教程的链接

**Reply by leotongxue** [reply]
我可以把我的电脑借你用😂 配的我都头疼死了

**Reply by Yang-Xijie** [reply]
Stack Overflow上现在应该有相关的调试教程了吧。我的教程应该只有调试不适用M1。
M1出来已经快一年了…

**Reply by leotongxue** [reply]
就是调试找不到相关的教程

**Reply by Levine66** [reply]
我手里的是M1 Pro 的Mac，遇到了无法正常debug的问题，解决办法是安装codeLLDB的插件，然后将launch.json中的type的属性值改为lldb即可，externalConsole改不改都行，最后的效果是只能用VS code中的终端，没有外部终端，可以正常调试。

**Reply by Urgotss** [reply]
你就是我的神,  之前教程里只能在终端调试,不能在vscode 里调试可急坏我了
